R/contribution_matrix.R

```diff
@@ -724,8 +724,12 @@ CreateContributionMatrix <- function(data, treatment_ids, outcome_type, outcome_
                                                treatments = treatments,
                                                outcome_type = outcome_type,
                                                outcome_measure = outcome_measure)$effect_sizes$Effect
-    #Multiply the n-th row of 'contribution' by the n-th element of 'effect_sizes'
-    contribution <- contribution * effect_sizes
```

Comment:
I don't understand why this doesn't do the job?
When I did the following:
```
matrix <- matrix(1:6, nrow = 2, ncol = 3)
test_effects <- c(2,3)
matrix * test_effects
```
This did indeed multiply the elements in the first row by 2, and the elements in the second row by 3.
Am I missing something?

Reply:
Sorry, I got myself confused. Each _column_ should be multiplied by a single treatment effect, not each row as I have written on Jira. I will change the text on Jira. The confusion is because of the transposing (again). It would be each row if the multiplication took place after transposing.